dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/Utils.java

```diff
@@ -127,6 +127,10 @@
         }
     }
 
+    public Pageable getPageable(@Nullable Pageable optionalPageable) {
```

Comment:
Could you add Javadocs to this new method?

Reply:
Good catch. Added javadoc.